pages/home/home.ejs

```diff
@@ -2,108 +2,512 @@
+                  <p class="card-text">
+                    Students are incentivized to repeat questions until mastery
+                  </p>
+                </div>
+              </div> <!-- card -->
+            </div> <!-- col -->
+          </div> <!-- row -->
+        </div> <!-- container -->
+        <div class="flare-students"></div>
+      </div>
 
-        <% } else { %> <!-- course_instances.length -->
+      <div class="position-relative bg-instructors text-center pt-5">
+        <div class="container">
+          <h2 class="pt-3 pb-5 text-uppercase font-weight-bold">Improve teaching workflows</h2>
+          <div class="row row-cols-1 row-cols-md-3">
+            <div class="col pb-5">
+              <div class="card bg-transparent text-center border-0">
+                <div class="card-img-top text-center display-2">
+                  <i class="fas fa-tasks text-dark" aria-hidden="true"></i>
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title">Adaptive</h3>
+                  <p class="card-text">
+                    Score adaptively based on student mastery and question difficulty
+                  </p>
+                </div>
+              </div> <!-- card -->
+            </div> <!-- col -->
+            <div class="col pb-5">
+              <div class="card bg-transparent text-center border-0">
+                <div class="card-img-top text-center display-2">
+                  <i class="fas fa-magic text-dark" aria-hidden="true"></i>
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title">Automated</h3>
+                  <p class="card-text">
+                    Automate the submission and grading process
+                  </p>
+                </div>
+              </div> <!-- card -->
+            </div> <!-- col -->
+            <div class="col pb-5">
+              <div class="card bg-transparent text-center border-0">
+                <div class="card-img-top text-center display-2">
+                  <i class="fas fa-chart-line text-dark" aria-hidden="true"></i>
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title">Analytics</h3>
+                  <p class="card-text">
+                    Analyze assessment quality and student outcomes
+                  </p>
+                </div>
+              </div> <!-- card -->
+            </div> <!-- col -->
+          </div> <!-- row -->
+        </div> <!-- container -->
+        <div class="flare-instructors"></div>
+      </div>
 
-        <table class="table table-sm table-hover table-striped">
-          <tbody>
-            <% course_instances.forEach(function(course_instance) { %>
-            <tr>
-              <td>
-                <a href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>">
-                  <%= course_instance.label %>
+      <div class="jumbotron bg-white my-3">
```

Comment:
At some point, links should point to relevant documentation... Though, it might be advantageous to design this to appear next to the in-browser editor for help?

Reply:
That would be awesome!